grade-scripts/trackers-from-requests.js

```diff
@@ -0,0 +1,160 @@
+const listManager = require('./shared/list-manager')
+const program = require('commander')
+const fs = require('fs')
+const execSync = require('child_process').execSync
+const chalk = require('chalk')
+const entityMap = require('../data/generated/entity-map')
+const utils = require('../src/utils')
+const scriptUtils = require('./shared/utils')
+
+const trackers = require('../src/trackers')
+const surrogates = require('../src/surrogates')
+
+program
+    .option('-i, --input <name>', 'The name to use when looking for sites, e.g. "test" will look in "test-sites"')
+    .option('-o, --output <name>', 'Output name, e.g. "test" will output files at "test-grades"')
+    .option('-f, --file <name>', 'Allow processing a subset of dumped site data, defined in a file')
```

Comment:
I wonder if the program options could be stored in scriptUtils, since they seem to be the same as for the other script above.
Also, again maybe it's a good idea to have default options